eks/ng/nodes.go

```diff
@@ -243,6 +243,10 @@ Resources:
         PropagateAtLaunch: true
       - Key: !Sub kubernetes.io/cluster/${ClusterName}
         Value: owned
+      - Key: !Sub k8s.io/cluster-autoscaler/${ClusterName}
```

Comment:
Can we make it as template? e.g. https://github.com/aws/aws-k8s-tester/blob/master/eks/ng/nodes.go#L211